README.md

```diff
@@ -1,5 +1,5 @@
 # hyprfreeze
-Hyprfreeze is a utility to suspend a game process (and other programs) in Hyprland.
+Hyprfreeze is a utility to suspend a game process (and other programs) on Wayland.
```

Comment:
"Hyprland and Sway" would be better instead of "Wayland" because there are still a lot more wayland compositors that are not supported

Reply:
I was going to keep it open (in case someone wants to provide support for other desktops). I guess it’s simple enough to change again if that happens :)